sql/core/src/test/scala/org/apache/spark/sql/streaming/ui/UISeleniumSuite.scala

```diff
@@ -51,6 +53,7 @@ class UISeleniumSuite extends SparkFunSuite with WebBrowser with Matchers with B
     val conf = new SparkConf()
       .setMaster(master)
       .setAppName("ui-test")
+      .set(SHUFFLE_PARTITIONS, 5)
```

Comment:
Just curious, is this to speed up the unit test not to start 200 tasks?

Reply:
Yes. Once I changed the active query a bit to have watermark being set, it suffered to make progress in 30 seconds (meaning checking the UI failed as there's no queryProgress) and failed. This fixed the issue.

Reply:
Has similar problem before, just wanted to double check. Thanks!